test/e2e/tests/account/migrate-old-vault.spec.ts

```diff
@@ -14,8 +15,13 @@ describe('Migrate vault with old encryption', function (this: Suite) {
       },
       async ({ driver }: { driver: Driver }) => {
         await loginWithBalanceValidation(driver);
-        const homePage = new HomePage(driver);
-        await homePage.headerNavbar.lockMetaMask();
+        const headerNavbar = new HeaderNavbar(driver);
+        await headerNavbar.check_pageIsLoaded();
```

Comment:
It seems waiting for this UI condition is not enough. The error somewhat indicates us it's a background related state not fully ready (snaps), so I'm hardcoding a bigger delay, which has shown is able to mitigate the issue completely